docker_daemon/conf.yaml.example

```diff
@@ -184,7 +184,8 @@ instances:
     #
     # container_tags: ["image_name", "image_tag", "docker_image"]
 
-    # List of container label names that should be collected and sent as tags.
+    # Legacy option for container label names that should be collected and sent as tags.
```

Comment:
You should add:
  - if they want to add high-card labels to your docker metrics but not your AD metrics, it's still a legit option
  - option will not be merged with the global one but replace it completely